poedit.appdata.xml

```diff
@@ -1,31 +1,76 @@
 <?xml version="1.0" encoding="utf-8"?>
-<components version="0.11">
-  <component type="desktop-application">
+<component type="desktop">
     <id>poedit.desktop</id>
     <name>Poedit</name>
     <summary>Translations editor for gettext</summary>
     <project_license>MIT</project_license>
+    <metadata_license>CC0</metadata_license>
     <description>
-      <p>
-        Translations editor for gettext (PO files). It helps with translating
-        applications or WordPress themes and plugins into other languages, as
-        well as with developing localizable applications.
-      </p>
+        <p>
+            Translations editor for gettext (PO files). It helps with translating
+            applications or WordPress themes and plugins into other languages, as
+            well as with developing localizable applications.
+        </p>
     </description>
     <screenshots>
-      <screenshot>
-        <image type="source">https://upload.wikimedia.org/wikipedia/commons/c/c2/Poedit_1.8.1_en.png</image>
-      </screenshot>
+        <screenshot>
+            <image type="source">https://upload.wikimedia.org/wikipedia/commons/c/c2/Poedit_1.8.1_en.png</image>
+        </screenshot>
     </screenshots>
     <url type="homepage">https://poedit.net</url>
+    <url type="bugtracker">https://github.com/vslavik/poedit</url>
     <url type="translate">https://crowdin.com/project/poedit</url>
+    <content_rating type="oars-1.0" />
+    <releases>
```

Comment:
I have mixed feelings about this. The usefulness of this is obvious, but I'd really like to avoid *manually* updating this information in two places (`NEWS` and this)...

Reply:
@vslavik the NEWS file is read by devs. The release tags here will be shown in the software center which will allow the users to see if it's worth it to upgrade the software to the latest release. You can of course only push the latest two releases here and so on. No need to keep the whole history.

Reply:
I am aware. The point is, `NEWS` is the canonical location for this information, it's a de-facto place for users (yes) to look for it, so I *have* to update that one. And I'd rather not manually write the same information in more than 1 place. "No need to keep the whole history" solves nothing in that regard (indeed, it's *more* work to remove old entries on top of adding new ones).

Reply:
You could somehow make the releases tags get updated automatically based on the NEWS file. I could create a simple script for that if you want and put it somewhere in the source tree.